src/layer/audio-source.ts

```diff
@@ -86,11 +86,16 @@ function AudioSourceMixin<OptionsSuperclass extends BaseOptions> (superclass: Co
 
       const load = () => {
         // TODO:              && ?
```

Comment:
This can go now. Not sure what I was thinking when I wrote that, but it no longer applies.